### ISSUE TITLE
Forestry trade station recipe missing from NEI (+ any with ore dictionary) - skmclauncher not loading NEI plugins

### ISSUE THREAD

**Issue by mushroomhostage** [mushroomhostage/exphc#67]
NEI shows the Trade Station machine from Forestry, but when you click it, it doesn't display the recipe.

This is made worse by the fact that http://forestry.sengir.net/wiki/index.php?n=Machines.TradeStation has the wrong recipe (should change: diamantine electron tube -> iron, and golden -> bronze).

And sometimes the Circuit Boards disappear from NEI, for no apparent reason, then reappear just as soon as they disappeared.

Appears to occur with or without the NEI Forestry plugin. See also https://github.com/mushroomhostage/exphc/issues/65

**Comment by mushroomhostage** [comment]
Some insight into this.. probably caused by new Forge ore dictionary recipes. Same problem with Iron Chests. QuickBench updated, though!

**Comment by mushroomhostage** [comment]
Fixed in 1.2.5rev4c, see https://github.com/mushroomhostage/exphc/wiki/1.2.5%20Server%20Setup

tl;dr: SK's Launcher wasn't loading NEI plugins.. new mod to do it: [LoadNEIPlugins](https://github.com/mushroomhostage/LoadNEIPlugins), + updated Forestry NEI plugin which also recognizes Forge Ore recipes. All works now, in this optional client-only update.

**Comment by mushroomhostage** [comment]
Posted about this LoadNEIPlugin to the SK's Launcher  thread here: http://www.minecraftforum.net/topic/1104030-launcher-sks-launcher-v315-easily-install-mods-update-mc-open-source/page__st__180#entry16668543
